src/Sign.Cli/Resources.resx

```diff
@@ -137,7 +137,7 @@
     <comment>{Locked="sha256", "sha384", "sha512"} are cryptographic hash algorithm names and should not be localized.</comment>
   </data>
   <data name="FileListOptionDescription" xml:space="preserve">
-    <value>Path to file containing paths of files to sign within an archive.</value>
+    <value>Path to file containing paths of files to sign within an archive or ClickOnce package.</value>
```

Comment:
I feel like we should be more generic, rather than be more specific.  It could also contain paths to DLL's, EXE's, NuGet packages, VSIX's, etc.